File: java/뒤에서 5등 위로.java
import java.util.*;

class Solution {
    public int[] solution(int[] num_list) {
        ArrayList<Integer> list = new ArrayList<>();
        Arrays.sort(num_list);
        for(int i = 5; i < num_list.length; i++) {
            list.add(num_list[i]);
        }
        int[] answer = list.stream().mapToInt(Integer::intValue).toArray();
        return answer;
    }
}

File: javascript/뒤에서 5등 위로.js
const solution = (num_list) => {
    let answer = [];
    const sorted_num_list = num_list.sort((a, b) => a - b);
    for(let i = 0; i < num_list.length; i++) {
        if(i >= 5) {
            answer.push(sorted_num_list[i]);
        }
    }
    return answer;
}